source/tutorials/learning-journey/sharing-dependencies.md

```diff
@@ -0,0 +1,204 @@
+# Sharing dependencies between `default.nix` and `shell.nix`
+
+<!-- Include any foreward you want here -->
+
+## Overview
+
+### What will you learn?
+
+In this tutorial you'll learn how not to repeat yourself by sharing dependencies between 'default.nix`, which is responsible for building the project, and `shell.nix`, which is responsible for providing you with an environment to work in.
+
+### How long will it take?
+
+This tutorial will take approximately 1 hour.
+
+### What will you need?
+
+This tutorial assumes you've seen a derivation (`mkDerivation`, `buildPythonApplication`, etc) before, and that you've seen `nix-shell` used to create shell environments.
```

Comment:
```suggestion
This tutorial assumes you're familiar with Nixpkgs build helpers (`mkDerivation`, `buildPythonApplication`, etc) and know how to create environments for `nix-shell`.
```

Ideally though this would just link to tutorials you should have done, if they exist.